client/src/main/kotlin/io/spine/examples/pingh/client/DesktopClient.kt

```diff
@@ -196,34 +196,12 @@ internal class DesktopClient(
             .observe(onUpdated)
             .post()
 
-    /**
-     * Subscribes to the update of the entity with the specified type and ID.
-     *
-     * The subscription cancels itself after the observer has completed its work.
-     *
-     * @param E The type of the observed entity.
-     *
-     * @param id The ID of the observed entity.
-     * @param type The class of the type of the observed entity.
-     * @param onUpdated Called when the entity is updated.
-     */
-    internal fun <E : EntityState> observeEntityOnce(
-        id: Message,
-        type: KClass<E>,
-        onUpdated: (entity: E) -> Unit
-    ): Subscription {
-        var subscription: Subscription? = null
-        subscription = observeEntity(id, type) { entity ->
-            stopObservation(subscription!!)
-            onUpdated(entity)
-        }
-        return subscription
-    }
-
     /**
      * Stops observation of the provided subscription.
+     *
+     * @param subscription The subscription to be discontinued.
      */
-    private fun stopObservation(subscription: Subscription) {
+    internal fun stopObservation(subscription: Subscription) {
```

Comment:
Let's rename to `cancel(...)`. And update the documentation accordingly.

"Stop observation", "discontinue" are both more complex for the reader's comprehension.